app.R

```diff
@@ -0,0 +1,1118 @@
+                           tabName = "instruction", 
+                           icon = icon("dashboard")
+                  ),
+                  menuItem(text = "Prerequisites", 
+                           tabName = "prereq", 
+                           icon = icon("book")
+                  ),
+                  menuItem(text = "Explore",
+                           tabName = "exploring", 
+                           icon = icon("wpexplorer")
+                  ),
+                  menuItem(text = "Game", 
+                           tabName = "game", 
+                           icon = icon("gamepad")
+                  ),
+                  menuItem(text = "References",
+                           tabName = "refs",
+                           icon = icon("leanpub")
+                  )
+      ),
+      tags$div(
+        class = "sidebar-logo", 
+        boastUtils::sidebarFooter()
+      )
+    ),
+    dashboardBody(
+      tabItems(
+        tabItem(tabName = "instruction",
+                h1("ANCOVA"),
+                h2("About:"),
+                p('This app introduces the concept of ANCOVA focusing on 
+                         interpreting interaction plots.'),
+                br(),
+                h2('Instructions:'),
+                tags$ul(
+                  tags$li('Click the prerequisite button to review the prerequisites
+                          if needed.'),
+                  tags$li('Navigate to the explore page. Use the dropdown menu to 
+                          select a dataset.'),
+                  tags$li('Use the radio buttons to select different variables 
+                                and see the changes in the interaction plot. You 
+                                can also use the slider bars to change the parameters.'),
+                  tags$li('After working with the explore section, you can 
+                                start the matching game to test your understanding 
+                                of the concepts.'),
+                  tags$li('Drag the items in each column so that the plots in the 
+                          first column are in the same order as their corresponding 
+                          output in the second column.'),
+                  tags$li('Then, click Submit to check your answers. Once all of 
+                          your answers are correct, click New for a new set of plots.')  
+                ),
+                div(style = "text-align: center",
+                    bsButton(inputId = "go",
+                             label = "Prerequisites",
+                             icon = icon("book"),
+                             style = "default",
+                             size = "large")
+                ),
+                br(),
+                h2('Acknowledgements:'),
+                p("This app was developed and coded by Luxin Wang and
+                         modified by Zhiruo Wang and Lydia Bednarczyk."),
```

Comment:
Maybe add the year you updated in because if someone else updates it later the update statement below will have their name and year